cypress/integration/metaPersianSpec.js

```diff
@@ -9,12 +10,16 @@ import {
   twitterMeta,
 } from '../support/metaTestHelper';
 
-describe('Persian Article Meta Tests', () => {
+skipLiveDescribe('Persian Article Meta Tests', () => {
   // eslint-disable-next-line no-undef
   before(() => {
     cy.visit(`/persian/articles/${config.assets.persian}`);
   });
 
+  it('should have the correct lang attribute', () => {
+    cy.get('html').should('have.attr', 'lang', 'fa');
```

Comment:
Can we make this config-driven?

I guess it does mean duplicating _exactly_ the same test in two files. But we can then refactor that further later.

Reply:
There currently isnt really any config place for this to go per service. We have per env config but not per service. I think this should be part of a greater piece of work in https://github.com/bbc/simorgh/issues/1371

Reply:
Agreed